packages/cbioportal-frontend-commons/src/components/checkedSelect/Checklist.tsx

```diff
@@ -103,22 +145,182 @@ export default class Checklist extends React.Component<ChecklistProps, {}> {
             return;
         }
 
-        // only select current value if all selected already
-        if (
-            this.props.unselectOthersWhenAllSelected &&
-            this.props.value.length === this.props.options.length
+        // splice protein type badge was selected
+        if (value === 'splice_site') {
```

Comment:
lets get rid of the special handling for splice_site and just edit the value to "spice" when we need to do.  that is, find/replace on value when you use it to compose these dynamic strings.